mitiq/shadows/classical_postprocessing.py

```diff
@@ -8,130 +8,360 @@
 # LICENSE file in the root directory of this source tree.
 """Classical post-processing process of classical shadows."""
 
-from typing import Any, List, Tuple
-
+from typing import Tuple, List, Any, Dict, Optional
 import cirq
 import numpy as np
 from numpy.typing import NDArray
-
 import mitiq
+from mitiq.shadows.shadows_utils import (
+    bitstring_to_eigenvalues,
+    create_string,
+    operator_ptm_vector_rep,
+    kronecker_product,
+)
 
+
+# local unitary that applied to the qubits
 phase_z = cirq.S._unitary_().conj()
 hadamard = cirq.H._unitary_()
 identity = cirq.I._unitary_()
 PAULI_MAP = {"X": hadamard, "Y": hadamard @ phase_z, "Z": identity}
 
+# z-basis measurement outcomes to 4d-vector representation
+b_zero = np.array([[1.0 + 0.0j, 0.0 + 0.0j]])
+zero_state = b_zero.T @ b_zero
+
+b_one = np.array([[0.0 + 0.0j, 1.0 + 0.0j]])
+one_state = b_one.T @ b_one
 
-def classical_snapshot(b_list: List[int], u_list: List[str]) -> NDArray[Any]:
+"""
+The following functions are used in the classical post-processing
+of calibration
+"""
+
+
+def get_single_shot_pauli_fidelity(
```

Comment:
It seems that you are now computing some kind of fidelity. But it is not clear what a `pauli-fidelity` is. The fidelity is usually  a _positive_ number measuring the overlap between a real quantum state and an ideal state. What are the real state and ideal state in this function?

Reply:
Update: I realized what the "pauli fidelity" is by reading the [rubust shadow paper](https://arxiv.org/abs/2011.09636).

I think it can be very helpful to add the following sentence in the docstring of this `get_single_shot_pauli_fidelity` function and also of the `get_pauli_fidelity` function:

`In the notation of arXiv:2011.09636, this function estimates the coefficients f_\gamma, which characterize the (noisy) classical shadow channel.`